generated/api.github.com.json

```diff
@@ -693,88 +638,16 @@
-      "url": "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/#permission-on-dependabot_secrets",
-      "read": [
-        "GET /orgs/{org}/dependabot/secrets",
-        "GET /orgs/{org}/dependabot/secrets/public-key",
-        "GET /orgs/{org}/dependabot/secrets/{secret_name}",
-        "GET /orgs/{org}/dependabot/secrets/{secret_name}/repositories",
-        "GET /repos/{owner}/{repo}/dependabot/secrets",
-        "GET /repos/{owner}/{repo}/dependabot/secrets/public-key",
-        "GET /repos/{owner}/{repo}/dependabot/secrets/{secret_name}"
-      ],
-      "write": [
-        "DELETE /orgs/{org}/dependabot/secrets/{secret_name}",
-        "DELETE /orgs/{org}/dependabot/secrets/{secret_name}/repositories/{repository_id}",
-        "DELETE /repos/{owner}/{repo}/dependabot/secrets/{secret_name}",
-        "PUT /orgs/{org}/dependabot/secrets/{secret_name}",
-        "PUT /orgs/{org}/dependabot/secrets/{secret_name}/repositories",
-        "PUT /orgs/{org}/dependabot/secrets/{secret_name}/repositories/{repository_id}",
-        "PUT /repos/{owner}/{repo}/dependabot/secrets/{secret_name}"
-      ]
-    },
-    "security_events": {
-      "url": "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/#permission-on-security-events",
-      "read": [
-        "GET /orgs/{org}/code-scanning/alerts",
-        "GET /repos/{owner}/{repo}/code-scanning/alerts",
-        "GET /repos/{owner}/{repo}/code-scanning/alerts/{alert_number}",
-        "GET /repos/{owner}/{repo}/code-scanning/alerts/{alert_number}/instances",
-        "GET /repos/{owner}/{repo}/code-scanning/analyses",
-        "GET /repos/{owner}/{repo}/code-scanning/analyses/{analysis_id}",
-        "GET /repos/{owner}/{repo}/code-scanning/sarifs/{sarif_id}"
-      ],
-      "write": [
-        "DELETE /repos/{owner}/{repo}/code-scanning/analyses/{analysis_id}",
-        "PATCH /repos/{owner}/{repo}/code-scanning/alerts/{alert_number}",
-        "POST /repos/{owner}/{repo}/code-scanning/sarifs"
-      ]
-    },
-    "self_hosted_runners": {
-      "url": "https://docs.github.com/en/free-pro-team@latest/rest/reference/permissions-required-for-github-apps/#permission-on-self-hosted-runners",
-      "read": [
-        "GET /orgs/{org}/actions/runners",
-        "GET /orgs/{org}/actions/runners/downloads",
-        "GET /orgs/{org}/actions/runners/{runner_id}",
-        "GET /orgs/{org}/actions/runners/{runner_id}/labels"
-      ],
-      "write": [
-        "DELETE /orgs/{org}/actions/runners/{runner_id}",
-        "DELETE /orgs/{org}/actions/runners/{runner_id}/labels",
-        "DELETE /orgs/{org}/actions/runners/{runner_id}/labels/{name}",
-        "POST /orgs/{org}/actions/runners/registration-token",
-        "POST /orgs/{org}/actions/runners/remove-token",
-        "POST /orgs/{org}/actions/runners/{runner_id}/labels",
-        "PUT /orgs/{org}/actions/runners/{runner_id}/labels"
-      ]
-    },
```

Comment:
`security_events` is added here: https://github.com/octokit/app-permissions/pull/142/files?show-viewed-files=true&file-filters%5B%5D=#diff-260b0feb344434d5b14f971a900a1ed6041cf8c0a5a393cd806a75277ce3c5c5R215

`self_hosted_runners` and `dependabot_secrets` are missing in schema